src/Options/docs.js

```diff
@@ -93,6 +94,17 @@
  * @property {String} webhookKey Key sent with outgoing webhook calls
  */
 
+/**
+ * @interface RateLimitOptions
+ * @property {String} errorResponseMessage The error message that should be returned in the body of the HTTP 429 response when the rate limit is hit. Default is `Too many requests.`.
+ * @property {Boolean} includeInternalRequests Optional, if `true` the rate limit will also apply to requests that are made in by Cloud Code, default is `false`. Note that a public Cloud Code function that triggers internal requests may allow to circumvent rate limiting and be vulnerable to a malicious attack. // Is this true? What exactly are "internal requests" here? Maybe we need to clarify more.
```

Comment:
The comment at the end of the line was meant for discussion, not be be committed